alma/alma-installments-prestashop
# feat: Insurance product page cannot be added to cart

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1833/[🧩-ps]-insurance-product-page)

### Code changes


We added hook to check if is an insurance product and remove the button add to cart

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Display the insurance product and see if you cannot add to cart the product

### Checklist for authors and reviewers




### Non applicable
